dotnet-authserver/tests/TeacherIdentity.AuthServer.Tests/EndpointTests/Admin/UserTests.cs

```diff
@@ -137,4 +138,41 @@ public async Task Get_ValidRequestForUserWithMergedUsers_RendersExpectedContent(
         var doc = await response.GetDocument();
         Assert.Equal(mergedUser.UserId.ToString(), doc.GetSummaryListValueForKey("Merged user IDs"));
     }
+
+    [Theory]
+    [InlineData(Models.TrnVerificationLevel.Low, "Low")]
+    [InlineData(Models.TrnVerificationLevel.Medium, "Medium")]
+    public async Task Get_ValidRequestForTrnVerificationLevel_RendersExpectedContent(Models.TrnVerificationLevel trnVerificationLevel, string verificationLevelString)
```

Comment:
Would be good to have another test that a user without a TRN doesn't show a row at all.